app/src/main/AndroidManifest.xml

```diff
@@ -1,54 +1,58 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
Did you change anything in this file or is it all just spacing?

Reply:
I altered it since I added a test activity then removed it, I'm gonna assume just a spacing thing